### ISSUE TITLE
Update template for Neighborhoods pages to remove MTF top logo

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-citylimits#147]

**Comment by benlk** [comment]
Example: https://citylimits.org/neighborhoods/east-harlem/

Present:

![Screen Shot 2020-02-04 at 18 05 50](https://user-images.githubusercontent.com/1754187/73795591-004eca80-4779-11ea-90e4-383738264e65.png)

To remove the image:

https://github.com/INN/umbrella-citylimits/blob/1287456c1b8474a782cfb020fcf32030d05b0d26/wp-content/themes/citylimits/header-rezone.php#L158

Removing the image from `header-rezone.php` will affect:

- `page-neighborhoods.php` - https://citylimits.org/zonein/
- `page-communitywire.php` - unsure; my db copy says post `2988252` which is https://citylimits.org/communitywire-home/ and a 404 on prod
- `page-neighborhood-info.php` - https://citylimits.org/zonein/affordable-housing-101/ as an example

Should this change only affect the Neighborhoods pages, or all pages currently using this header?

After removing the image:

![Screen Shot 2020-02-04 at 18 06 24](https://user-images.githubusercontent.com/1754187/73795633-178db800-4779-11ea-8834-017dc6d438bb.png)

Are there any other changes that need to be made to this area?

**Comment by MirandaEcho** [comment]
@benlk - its an old logo, so lets remove it from anywhere its used in the header

**Comment by MirandaEcho** [comment]
What would it take to add the top portion (name, photo, blurb, icon) from this page: https://citylimits.org/series/futuremap/ to the Neighborhoods template as well?

**Comment by joshdarby** [comment]
> What would it take to add the top portion (name, photo, blurb, icon) from this page: https://citylimits.org/series/futuremap/ to the Neighborhoods template as well?

@MirandaEcho Do you want an estimate for that or do you want me to try and cram it in to the remaining 57m this issue has?

**Comment by joshdarby** [comment]
Update: tried adding the section to the old template while using the existing CSS. Most of it worked, but we'd need to modify some parts. I'd think we could get this done in under an hour and a half (including review).

**Comment by joshdarby** [comment]
Another update:

The only modification I had to make was moving `#series-header` and `.series-banner` out of the parent `.cftl-tax-landing-template-series-landing-special-projects-php` styling

![screencapture-citylimits-test-series-futuremap-2020-02-19-15_31_34-1](https://user-images.githubusercontent.com/18353636/74873458-379ea900-532d-11ea-9078-6ece0fa8752f.png)

The only issue I can see is that from 800px to 600px, the header area overlaps the `rezone-overview` area

![Screen Shot 2020-02-19 at 3 34 00 PM](https://user-images.githubusercontent.com/18353636/74873484-44bb9800-532d-11ea-98f2-0ae5108bd4a2.png)

**Comment by MirandaEcho** [comment]
@joshdarby - is this done then and on staging?

**Comment by joshdarby** [comment]
@MirandaEcho No, I was never told whether or not the estimate I gave was approved and that I could finish up the work on it

**Comment by MirandaEcho** [comment]
I wasn't sure if that last update meant it was done or not, sorry! Please proceed. Thanks!

**Comment by joshdarby** [comment]
@MirandaEcho This is now deployed to staging.

**Comment by MirandaEcho** [comment]
@joshdarby lets hardcode this as the featured image in the top (next to the page title) while keeping the neighborhood featured image below it:

![Mapping the future](https://user-images.githubusercontent.com/48263361/75477254-a13c3a00-5961-11ea-9d20-dfb03fb5156c.jpg)
